src/content/quickstart/kind/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2020-03-17T13:36:18+01:00
+title: "KIND"
+---
+
+## KIND
+
+[KIND](https://github.com/kubernetes-sigs/kind) is a tool to run local Kubernetes clusters inside Docker container nodes.
+Submariner provides with scripts that deploy 3 Kubernetes clusters locally, 1 broker and 2 data clusters with submariner
+running on both the data clusters.
+
+To deploy the setup, clone [submariner repo](https://github.com/submariner-io/submariner) and run
+
+`make ci e2e status=keep`
+
+This will deploy 3 Kubernetes clusters and run basic [end-to-end](https://github.com/submariner-io/submariner/tree/master/test/e2e) tests.
+
```

Comment:
I'm wondering if we should say something about `status=keep` - perhaps "_The status=keep option retains the cluster setup after the tests complete_"? Or maybe just omit it?

Reply:
I didn't add that thinking this is a quickstart and the details can be read on the link provided. But no harm in adding.

Reply:
yeah  that's why I also suggested omitting `status=keep`.